spring-cloud-gcp-pubsub-stream-binder/src/test/java/com/google/cloud/spring/stream/binder/pubsub/PubSubEmulator.java

```diff
@@ -53,7 +54,7 @@
  *
  * @since 1.1
  */
-public class PubSubEmulator implements BeforeAllCallback, AfterAllCallback {
+public class PubSubEmulator implements BeforeAllCallback, AfterAllCallback, TestInstancePostProcessor {
```

Comment:
I think it might be simpler to instead implement `ParameterResolver`, and return _itself_ as an instance of `PubSubEmulator`. Then the test can accept `PubSubEmulator` as a constructor argument and continue to function without any new functionality (emulator instance in  used to be static, but I don't think it needs to be anymore now that it's not a class rule).

Reply:
Hmm, good point. Let me try this.